quickstarts/c#/PersonalizerExample/NuGet.Config

```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-    <packageSources>
-        <add key="personalization-rl repo" value="../../nuget" />
-    </packageSources>
+  <packageSources>
+    <add key="personalizer-rl repo" value="../../../preview-nuget" />
```

Comment:
This text doesn't make sense because the sample has been moved to the cognitive-services-personalizer-samples now. Maybe "Local Respository" or "cognitive-services-personalizer-samples repository"

Reply:
agree

Reply:
fixed